frontend/src/components/Bootstrap/index.tsx

```diff
@@ -19,26 +21,33 @@ type Form = {
 
 type State = {
   isInProgress: boolean;
+  isSubmitInProgress: boolean;
```

Comment:
Can't we use `isSubmitting` from react-hook-form's `formState`? https://www.react-hook-form.com/api/useform/formstate/